spec/request/organization_quotas_spec.rb

```diff
@@ -462,6 +462,20 @@ module VCAP::CloudController
           expect(last_response).to include_error_message("Organization Quota '#{organization_quota.name}' already exists.")
         end
       end
+
+      context 'when trying to set a log rate limit and there are apps with unlimited log rates' do
+        let!(:app_model) { VCAP::CloudController::AppModel.make(name: 'name1', space: space) }
+        let!(:process_model) { VCAP::CloudController::ProcessModel.make(app: app_model, log_rate_limit: -1) }
+
+        it 'returns 422' do
+          patch "/v3/organization_quotas/#{organization_quota.guid}", params.to_json, admin_header
+
+          expect(last_response).to have_status_code(422)
+          expect(last_response).to include_error_message(
+            'Current usage exceeds new quota values. ' \
+            "Org '#{org.name}' assigned this quota contains apps running with an unlimited log rate limit.")
```

Comment:
This error message is upon first read. Maybe adding commas would help?

Reply:
I'm not sure. We did struggle a little with the wording here (and varying the messages based on the number of affected orgs).

Can you provide an alternative error message that you think is less confusing?

Reply:
I agree theres no good way to word it. Personally I find one of the following _slightly_ easier to understand.

"Org '#{org.name},' assigned this quota, contains apps running with an unlimited log rate limit."
or
"This quota is applied to org '#{org.name}' which contains apps running with an unlimited log rate limit."

Reply:
@MerricdeLauney Thanks. I pushed a commit to reword following the second option you suggested.